yt/frontends/artio/_artio_caller.pyx

```diff
@@ -14,14 +14,10 @@ from libc.string cimport memcpy
 from yt.geometry.oct_container cimport OctObjectPool, SparseOctreeContainer
 from yt.geometry.oct_visitors cimport Oct
 from yt.geometry.particle_deposit cimport ParticleDepositOperation
-from yt.geometry.selection_routines cimport (
-    AlwaysSelector,
-    OctreeSubsetSelector,
-    SelectorObject,
-)
+from yt.geometry.selection_routines cimport AlwaysSelector, SelectorObject
 from yt.utilities.lib.fp_utils cimport imax
 
-import data_structures
+#import data_structures
```

Comment:
Should it be removed altogether?
```suggestion
```

Reply:
Yes, I initially just commented it because these modules have side effects so importing them may be legit even if "unused", but it doesn't seem to matter here